rust/sdk/Cargo.lock

```diff
@@ -1138,6 +1231,18 @@ version = "0.8.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "34af8d1a0e25924bc5b7c43c079c942339d8f0a8b57c39049bef581b46327404"
 dependencies = [
+ "libc",
+ "rand_chacha",
```

Comment:
curious, this is the same package with the same checksum, how do its dependencies change?

Reply:
Features (maybe)